cirq-google/cirq_google/api/v2/program.proto

```diff
@@ -434,4 +437,24 @@ message CouplerPulseGate{
   optional FloatArg coupling_mhz = 4;
   optional FloatArg q0_detune_mhz = 5;
   optional FloatArg q1_detune_mhz = 6;
+}
+
+message CliffordTableau {
+  optional int32 num_qubits = 1;
+  optional int32 initial_state = 2;
+  optional bytes rs = 3;
```

Comment:
Same comment as before w.r.t. numpy serialization.  In this case, I think it might be okay, but we should at least document it.

Reply:
I changed the serialization to be independent of numpy